.github/workflows/test-integration-runner.yaml

```diff
@@ -388,10 +388,10 @@ jobs:
 
             # Dynamic version-specific external secret configuration
             APP_VERSION=$(helm show chart "charts/${{ inputs.camunda-helm-dir }}" | yq .appVersion | grep -o '^[0-9]*\.[0-9]*')
-            if [[ -f ".github/config/external-secret/external-secret-${APP_VERSION}-integration-test-credentials.yaml" ]]; then
+            if [[ -f "charts/${{ inputs.camunda-helm-dir }}/test/external-secret/external-secret-${APP_VERSION}-integration-test-credentials.yaml" ]]; then
```

Comment:
No need for the `${APP_VERSION}` here, inside the chart versions, no need for the version again.

So it should be just `external-secret-integration-test-credentials.yaml`.

Also, the `external-secret` should be inside the [integration](https://github.com/camunda/camunda-platform-helm/tree/89b4d3ec1a3a4ceb64adce4d3ba73eaa7693cb3c/charts/camunda-platform-8.8/test/integration) dir because those secrets are only used in the integration tests.